test/run.d

```diff
@@ -143,6 +143,13 @@ Options:
         return spawnProcess(unitTestRunnerCommand ~ args).wait();
     }
 
+    if (args == ["tools"])
+    {
+        verifyCompilerExists(env);
+        ensureToolsExists(env, EnumMembers!TestTools);
+        return 0;
+    }
```

Comment:
This allows to prebuild the test tools via `run tools`, which simplifies some scenarios, where e.g. environment variables are needed for the DMD/optlink/... to be tested, but would also affect the host compiler.